Error : No rule to make target 'build-c'

**Describe the bug**
Can not build bootloader

**Environment information:**

- OS: Linux,
- Platform: ?
- Firmware Version: 453e7805e213367335dbe54fa5a0e7149563c4ce

**Steps to Reproduce**

After following instructions in README

```
$ git clone https://github.com/skycoin/hardware-wallet
$ cd hardware-wallet
$ make bootloader
```

**Actual behavior**

```
make -C tiny-firmware/protob/ build-c
make[1]: Entering directory '/home/hardware-wallet/tiny-firmware/protob'
make[1]: *** No rule to make target 'build-c'.  Stop.
make[1]: Leaving directory '/home/hardware-wallet/tiny-firmware/protob'
Makefile:96: recipe for target 'build-deps' failed
make: *** [build-deps] Error 2
```

**Expected behavior**
Build firmware successfully .

**Additional context**
Running inside Docker container . Originally reported by @jdknives

--- a/FAQ.md
+++ b/FAQ.md
@@ -28,3 +28,46 @@
 If you see an inverted skycoin logo in bootloader mode like in the following image, maybe you have either flashed for development or not official device.
 
 ![Kiku](images/skycoin_logo_inverted_bootloader.jpg)
+
+### What should I do if something fails?
+
+If you are experiencing any issues:
+
+- Determine if your particular case is mentioned in [troubleshooting section](#troubleshooting) and try to follow the corresponding instructions to fix it.
+- Contact the developers via [Skycoin development channel in Telegram](https://t.me/skycoindev).
+
+## Troubleshooting
+
+This section describes quick solutions to some common errors or mistakes.
+
+### What does "Got TypeError when importing the protocol definitions for generator" message mean?
+
+If you are getting this error quite likely your system is configured to run Python version `2.x` by default. This version of Python is not supported. At all times `Python3` is a requirement.
+
+Firstly , confirm that Python 2.x was executed by running the following commands from a terminal window:
+
+```sh
+which protoc
+protoc --version
+python -c 'import google.protobuf; print(google.protobuf.__file__)'
+```
+
+If `python2` is part of the output then that's exactly the case so continue with instructions that follow. Otherwise [contact the team](#what-should-i-do-if-something-fails).
+
+In order to force using Python3 set `PYTHON=python3` environment variable while invoking all make targets. For instance if building bootloader binary just once
+
+```sh
+make bootloader PYTHON=python3
+```
+
+In order to invoke multiple targets export that variable to the glovbal OS environment, e.g. on Unix systems
+
+```sh
+export PYTHON=python3
+make bootloader
+make firmware
+make full-firmware
+```
+
+For further details see #235 .
+

--- a/README.md
+++ b/README.md
@@ -57,7 +57,7 @@ Follow the instructions written on [tiny-firware/README.md](https://github.com/s
 
 ## Build instructions:
 
-Immediately after cloning this repository make sure submoudules are up-to-date by executing. All other commands expect this step completed first.
+Immediately after cloning this repository make sure submoudules are up-to-date by executing the following command. All other commands expect this step completed first.
 
 ```
 git submodule update --init --recursive

--- a/README.md
+++ b/README.md
@@ -57,13 +57,17 @@ Follow the instructions written on [tiny-firware/README.md](https://github.com/s
 
 ## Build instructions:
 
-### Build and run emulator
-
-Update submodules:
+Immediately after cloning this repository make sure submoudules are up-to-date by executing. All other commands expect this step completed first.
 
 ```
 git submodule update --init --recursive
+```
 
+Should you find any issues while running any of the commands that follow please consult [FAQ](FAQ.md) before [reporting a bug](ihttps://github.com/skycoin/hardware-wallet/issues/new?assignees=&labels=bug&template=bug_report.md&title=).
+
+### Build and run emulator
+
+```
 make clean && make run-emulator
 ```
 

